client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -34,26 +35,29 @@ function EditorDrawerTaxonomies( { siteId, postType, postTerms, taxonomies } ) {
 						<Accordion
 							key={ taxonomy.name }
 							title={ taxonomy.label }
-							icon={ <Gridicon icon="category" /> }
+							icon={ <Gridicon icon="folder" /> }
 						>
-							Work in Progress
-						</Accordion>
-					);
-				} else {
-					return (
-						<Accordion
-							key={ taxonomy.name }
-							title={ taxonomy.label }
-							icon={ <Gridicon icon="tag" /> }
-						>
-							<TermTokenField
-								postTerms={ postTerms }
+							<TermSelector
+								selectedTermIds={ postTerms ? map( postTerms[ taxonomy.name ], 'ID' ) : [] }
 								taxonomyName={ taxonomy.name }
-								taxonomyLabel={ taxonomy.label }
 							/>
 						</Accordion>
 					);
 				}
+
+				return (
+					<Accordion
+						key={ taxonomy.name }
+						title={ taxonomy.label }
+						icon={ <Gridicon icon="tag" /> }
+					>
+						<TermTokenField
+							postTerms={ postTerms }
```

Comment:
These components are already pretty tightly tied to editing posts - [recording stats, `PostActions.edit`](https://github.com/Automattic/wp-calypso/blob/d98ba96676263eb9f1b6bc8d42c912af499a6d02/client/post-editor/term-token-field/index.jsx#L28-L47), and [pulling terms from a post terms object](https://github.com/Automattic/wp-calypso/blob/d98ba96676263eb9f1b6bc8d42c912af499a6d02/client/post-editor/term-token-field/index.jsx#L49-L61).  Take that away and you're left with just `TokenField`, so a component for another purpose should be written from scratch.